osu.Game/Screens/Ranking/Expanded/Statistics/PerformanceStatisticTooltip.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Extensions.LocalisationExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Cursor;
+using osu.Framework.Graphics.Shapes;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets.Difficulty;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Ranking.Expanded.Statistics
+{
+    public class PerformanceStatisticTooltip : VisibilityContainer, ITooltip<PerformanceBreakdown>
+    {
+        private readonly Box background;
+        private Colour4 titleColor;
+        private Colour4 textColour;
+
+        protected override Container<Drawable> Content { get; }
+
+        public PerformanceStatisticTooltip()
+        {
+            AutoSizeAxes = Axes.Both;
+            Masking = true;
+            CornerRadius = 5;
+
+            InternalChildren = new Drawable[]
+            {
+                background = new Box
+                {
+                    RelativeSizeAxes = Axes.Both
+                },
+                Content = new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Vertical,
+                    Padding = new MarginPadding { Left = 10, Right = 10, Top = 5, Bottom = 5 }
+                },
+            };
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            background.Colour = colours.Gray3;
+            titleColor = colours.Blue;
+            textColour = colours.BlueLighter;
+        }
+
+        protected override void PopIn()
+        {
+            // Don't display the tooltip if "Total" is the only item
+            if (currentPerformance.Performance.GetAttributesForDisplay().Count() > 1)
```

Comment:
This conditional should really live in `SetContent()`. `PopIn()` shouldn't contain actual logic if it can be avoided, just the transition definition.